tasks/Module 5/task_2.py

```diff
@@ -0,0 +1,233 @@
+            row[item_position] = GameItem(item_name, randint(15, 20), cost, heal)
+
+    def move_hero(self, hero, direction):
+        """Move hero."""
+        initial_position = hero.position
+        # print('initial position', initial_position)
+        if direction in ("north", "u"):
+            # to not cross border
+            aligned_init_pos = (
+                len(self.field) if initial_position[0] == 0 else initial_position[0]
+            )
+            # get current contenf of the cell
+            cell_content = self.field[aligned_init_pos - 1][initial_position[1]]
+            # move hero to new position
+            self.field[aligned_init_pos - 1][initial_position[1]] = HERO_SYMBOL
+            # change hepo position on hero object
+            hero.position = (aligned_init_pos - 1, initial_position[1])
+        elif direction in ("south", "d"):
+            aligned_init_pos = (
+                -1
+                if initial_position[0] == len(self.field) - 1
+                else initial_position[0]
+            )
+            cell_content = self.field[aligned_init_pos + 1][initial_position[1]]
+            self.field[aligned_init_pos + 1][initial_position[1]] = HERO_SYMBOL
+            hero.position = (aligned_init_pos + 1, initial_position[1])
+        elif direction in ("west", "l"):
+            aligned_init_pos = (
+                len(self.field[0]) if initial_position[1] == 0 else initial_position[1]
+            )
+            cell_content = self.field[initial_position[0]][aligned_init_pos - 1]
+            self.field[initial_position[0]][aligned_init_pos - 1] = HERO_SYMBOL
+            hero.position = (initial_position[0], aligned_init_pos - 1)
+        elif direction in ("east", "r"):
+            aligned_init_pos = (
+                -1
+                if initial_position[1] == len(self.field[0]) - 1
+                else initial_position[1]
+            )
+            cell_content = self.field[initial_position[0]][aligned_init_pos + 1]
+            self.field[initial_position[0]][aligned_init_pos + 1] = HERO_SYMBOL
+            hero.position = (initial_position[0], aligned_init_pos + 1)
+        # clear initial hero position
+        self.field[initial_position[0]][initial_position[1]] = " "
+        # print('position after move', hero.position)
+        return cell_content
+
+    def __str__(self):
+        """Draw field."""
+        to_print = ""
+        for _, row in enumerate(self.field):
+            to_print += "+---" * self.columns + "+\n"
+            to_print += (
+                "| "
+                + " | ".join(map(lambda x: ITEM_SYMBOL if isinstance(x, Item) else x, row))
+                + " |\n"
+            )
+        to_print += "+---" * self.columns + "+\n"
+        return to_print
+
+
+class Game:
+    """Main Game class."""
+
+    def __init__(self, columns, rows, hero):
+        """Init Game instance."""
+        self.field = Board(columns, rows)
```

Comment:
Now it should be `self.board` instead of `self.field`. Or at least `self.fields`.